ansible.cfg

```diff
@@ -4,6 +4,7 @@ stdout_callback=debug
 stderr_callback=debug
 roles_path=roles
 collections_paths=collections
+private_key_file=~/.ssh/galaxy
```

Comment:
Could you please remove this change - we have some VMs that need a different key so it's not convenient to declare the path in ansible.cfg

Reply:
This will be overridden by the `--key-file` param - do those VMs use this, or do they default to an `id_rsa` key?

Reply:
I understand that but there are also contexts where key file does not have to be defined at all and the ssh key is already loaded (i.e. with jenkins) so pointing to a path that does not exist is inconvenient

Reply:
No worries, will remove now